include/bootloader/core/ids.h

```diff
@@ -216,6 +216,7 @@ typedef enum {
     can_movestopcondition_gripper_force = 0x8,
     can_movestopcondition_stall = 0x10,
     can_movestopcondition_ignore_stalls = 0x20,
+    can_movestopcondition_limit_switch_backoff = 0x30,
```

Comment:
Can we change this to `0x40` so the enum can be treated as a bitmask?

Reply:
yes yes it should all be 0x40, forgot to change this one